package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "s3",
-  "version": "7.10.31",
+  "version": "7.10.32",
```

Comment:
Can you do a separate commit for the version bump? It helps when it comes to backporting things (as well as keeping track of version changes more easily).

Reply:
Done, I agree. I usually split it.